internal/manifests/manifestutils/labels.go

```diff
@@ -18,28 +18,36 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/go-logr/logr"
```

Comment:
the API imports in this file are not ordered. the `metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"` should be here